web/api/v1/api.go

```diff
@@ -768,6 +768,9 @@ type Target struct {
 	LastScrape         time.Time           `json:"lastScrape"`
 	LastScrapeDuration float64             `json:"lastScrapeDuration"`
 	Health             scrape.TargetHealth `json:"health"`
+
+	Interval string `json:"interval"`
```

Comment:
Can we call that scrape_interval and scrape_timeout, like the config options?

Reply:
I was thinking that after validating in `populateLabels`, we could then modify the actual config fields, which would allow us to access the values here and `sync` without any additional parsing. WDYT?

Reply:
yes that looks like a simple approach, it makes populateLabels a bad name then, but I do not have a better name. let's make sure that this is documented in populateLabels help.

Reply:
After trying to actually implement this, I've realized that it's not viable because the config passed to `populateLabels` is for the entire target group, not just the individual targets which we want to modify. This leaves us with two options:

1. Keep the existing system where we store the interval and timeout in labels and re-parse to access using `IntervalAndTimeout`. This has the benefit of complying with how we store other values like `job`, `address`, and `scheme`.
2. Store the values in standalone fields in the `Target`. This way, we don't have to parse them again when accessing.

Reply:
Let's go for 1.